webapp/services/import_services/ocpi/ocpi_validators.py

```diff
@@ -125,7 +126,7 @@ class GeoLocationInput(ValidataclassMixin):
 @validataclass
 class DisplayTextInput(ValidataclassMixin):
     language: str = StringValidator(min_length=2, max_length=2)
-    text: str = StringValidator(max_length=512)
+    text: str = ReplacingStringValidator(mapping={'\r': '', '\n': ' ', '\xa0': ' '}, max_length=512)
```

Comment:
'\r' is removed, but ' \n' and '\xa0' are substituted by a white space. Is this how we need it to be?

Reply:
It think that's useful for readability - `\r`usually doesn't occur on its own, and replacing a line break with a space avoids the words/sentences sticking weirdly together in the resulting text. `\xa0` is a space anyway, we just replace it with a 'normal' space to avoid the trouble that might come with handling non-printable characters.

Reply:
At further thought... it might still be helpful to squash multiple spaces together after that, in case there were already some spaces after the newlines. 
@the-infinity I remember there being some places where multiple spaces are used intentionally, to enforce proper line breaks at a charge station display with fixed line length. Is that something that could ever be relevant for imported data in OCPDB or can I safely discard the extra spaces?

Reply:
This will never be a thing at the OCPDB, as the OCPDB does not handle Authorizations, so multiple spaces can safely be replaced by a single space.